custom-sign-in/app/build.gradle

```diff
@@ -10,10 +10,10 @@ android {
         versionName "1.0"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
 
-        buildConfigField("String", "BASE_URL", "\"{org-domain-with-https-schema}\"")
+        buildConfigField("String", "BASE_URL", "\"https://{yourOktaDomain}\"")
         manifestPlaceholders = [
-                hostName:"{only-org-domain}",
-                "appAuthRedirectScheme": "{redirect_uri_schema}"
+                hostName:"{yourOktaDomain}",
```

Comment:
I wonder if we can derived `hostName` from `BASE_URL` or other way around?

Reply:
we can simplify this sample to not need this hostname